### PR TITLE
Calling events when players are removed from plots & correct calling of Member & Trusted events

### PR DESCRIPTION
**Pull request by MisterErwin**
Now calling the respective events when a user is removed from a plot.

And switched the event calling of adding Helpers and Trusted players. (Correct me if that was wrong)
#1300

### CODE REVIEW

File: Bukkit/src/main/java/com/plotsquared/bukkit/util/BukkitEventUtil.java [hunk 1]
```diff
@@ -1,30 +1,9 @@
...
 
 import com.intellectualcrafters.plot.flag.Flag;
-import com.intellectualcrafters.plot.object.Location;
-import com.intellectualcrafters.plot.object.Plot;
-import com.intellectualcrafters.plot.object.PlotArea;
-import com.intellectualcrafters.plot.object.PlotCluster;
-import com.intellectualcrafters.plot.object.PlotId;
-import com.intellectualcrafters.plot.object.PlotPlayer;
-import com.intellectualcrafters.plot.object.Rating;
+import com.intellectualcrafters.plot.object.*;
```
Comment (manuelgu) [inline Bukkit/src/main/java/com/plotsquared/bukkit/util/BukkitEventUtil.java]: Don't use * imports.

File: Bukkit/src/main/java/com/plotsquared/bukkit/util/BukkitEventUtil.java [hunk 2]
```diff
@@ -1,30 +1,9 @@
...
-import com.plotsquared.bukkit.events.PlayerTeleportToPlotEvent;
-import com.plotsquared.bukkit.events.PlotClearEvent;
-import com.plotsquared.bukkit.events.PlotComponentSetEvent;
-import com.plotsquared.bukkit.events.PlotDeleteEvent;
-import com.plotsquared.bukkit.events.PlotFlagAddEvent;
-import com.plotsquared.bukkit.events.PlotFlagRemoveEvent;
-import com.plotsquared.bukkit.events.PlotMergeEvent;
-import com.plotsquared.bukkit.events.PlotRateEvent;
-import com.plotsquared.bukkit.events.PlotUnlinkEvent;
+import com.plotsquared.bukkit.events.*;
```
Comment (manuelgu) [inline Bukkit/src/main/java/com/plotsquared/bukkit/util/BukkitEventUtil.java]: Same here.

**Comment by boy0001** [comment]
@MisterErwin Legacy reasons, PlotSquared when the groups were different. You'll see other references to Trusted/Helpers instead of Members/Trusted. 

I see no problem merging this for the 3.5 update, unless @MattBDev has something to add to the conversation?

**Comment by MattBDev** [comment]
As long as it is a non-breaking change, I'm fine with it.

**Comment by MisterErwin** [comment]
Till now nothing broke on my test server. It appears to be fine on the "production server",

**Pull request merged**